Documentation/Signoffs/Start Sensor/Start_Sensor_Detail_Design.md

```diff
@@ -0,0 +1,127 @@
+
+![Alt text](https://github.com/cebttu/CapstoneTeam1/blob/CazBilbrey-StartSensor-SignOff/Documentation/Signoffs/Start%20Sensor/3D%20Model%20Side.jpg)
+<br />
+Figure 4. Precise Height of Start Sensor
+
+Figure 4 (above) shows the scale of how the Sensor and button will be located from a side view. You can see that the bottom of the sensor and button range from 3.7-4.3 inches due to the location of the Green LED placed on the course (Figures 6 & 7). The height of the tracks, once designed in the Drivetrain System, will adjust this height further, however the sensor must hit this range to properly read the green light that indicates the start of the run(Constaint 4).
+
+![Alt text](https://github.com/cebttu/CapstoneTeam1/blob/CazBilbrey-StartSensor-SignOff/Documentation/Signoffs/Start%20Sensor/3D%20Model%20Top.png)
+<br />
+Figure 5. Precise location of Start Sensor
+
+Figure 5 (above) shows where the Sensor will lay out on the Robot and a rough location of the button. You can read more about the button location in the analysis section. This does not include the perf board because it allows for a cleaner view of what is happening.
+
+![Alt text](https://github.com/cebttu/CapstoneTeam1/blob/CazBilbrey-StartSensor-SignOff/Documentation/Signoffs/Start%20Sensor/button%20with%20perf%20board.PNG)
+<br />
+Figure 6. Button and Sensor attached to Robot
+
+Attaching to Chassis:
+
+Figure 6 offers a rough visual representation of how the Button and Sensor are integrated into the robot. Our strategy involves employing a perf board for the button, which will be securely soldered into position, while the sensor will be affixed to the chassis using screw fastenings. This assembly method is designed for maximum stability and reliability in our robot's configuration, as exemplified by the corner bolts that secure the boards firmly in place. 
+
+## Analysis
+
+Constraint 1 Solution:  The RGB Sensor shall detect a spectrum of green light from an LED. Green LEDs typically output light at wavelengths of 570nm to 575nm. The sensor detects the percentage of the red light, blue light, and green light detected in the color it is receiving and outputs this to the main controller via the I2C bus. The main controller will take these percentages of red, green, and blue and will determine if it fits within the range of percentages generated by a wavelength of 570nm to 575nm. 
+
+Constraint 2 Solution: The RGB Sensor shall detect the light within 3 seconds of the light being turned on. The RGB Sensor is crucial for our competition performance as it's required to read and output within a strict 3-second timeframe. This timeframe is reasonable as the chip can convert analog signals to digital signals in a typical value of 2.78 ms and the I2C bus can begin to transmit this signal in a minimum of 2.03 us. This gives plenty of time for the sensor to detect the light, convert the signal, and send data within 3 seconds.
```

Comment:
These times don't seem realistic or meaningful. What does analog to digital conversion have to do with anything of this design?